gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
 version=2.2.2-SNAPSHOT
-ballerinaLangVersion=2201.0.1
+ballerinaLangVersion=2201.0.4
```

Comment:
Do we need to change this as well?

Reply:
I thought since we are moving into `2201.0.4` distribution, there is no harm in moving into `2201.0.4` lang version as well. @niveathika any thoughts on this ?

Reply:
As per the discussion I will revert this change.

Reply:
Fixed with [b80d00dd270f25087b7f8a5c560516e6b79f7065](https://github.com/ballerina-platform/module-ballerina-websub/pull/533/commits/b80d00dd270f25087b7f8a5c560516e6b79f7065)